Formula/cava.rb

```diff
@@ -0,0 +1,41 @@
+class Cava < Formula
+  desc "Console-based Audio Visualizer for ALSA"
+  homepage "https://github.com/karlstav/cava"
+  url "https://github.com/karlstav/cava/archive/0.7.4.tar.gz"
+  sha256 "fefd3cc04d41b03ca416630cafadbfda6c75e2ca0869da1f03963dcb13e1ecb7"
+  license "MIT"
+  head "https://github.com/karlstav/cava.git", branch: "master"
+
+  depends_on "automake"
+  depends_on "libtool"
+  depends_on "fftw"
+  depends_on "iniparser"
+  depends_on "portaudio"
+
+  uses_from_macos "ncurses"
+
+  def install
+    # force autogen.sh to look for and use our glibtoolize
+    inreplace "autogen.sh", "libtoolize", "glibtoolize"
+    # correct Makefile.am with hardcoded libpath
+    inreplace "Makefile.am", "/usr/local", "#{HOMEBREW_PREFIX}"
```

Comment:
This won't help and will just cause `brew style` errors.
```suggestion
    inreplace "Makefile.am", "/usr/local", HOMEBREW_PREFIX
```
What you need is a build dependency on `autoconf` since `autogen.sh` is looking for `autom4te` after `Makefile.am` has been modified.

Reply:
I changed that.

The thing with the build dependency: I tried that. But it fails. With a "standard" dependency it nevertheless works...

Reply:
If you add `:build` dependency on `autoconf` it will work.

Reply:
ahh okay. :build dependencies have to list piece by piece... I thought since automake has a dependency on autoconf, all dependencies are there...